test/rekt/apiserversource_test.go

```diff
@@ -67,8 +67,8 @@ func TestApiServerSourceValidationWebhookConfigurationOnUpdate(t *testing.T) {
 }
 
 func TestApiServerSourceDataPlane_SinkTypes(t *testing.T) {
-	t.Parallel()
-
+	//t.Parallel()
+	t.Skip("Skipping until the test logic is fixed.")
```

Comment:
do we track this somewhere ? e.g. in a GH issue?

Reply:
Related to issue #7180.
Since I addressed different problems in separate PRs and #7180 has already been merged, I believe this test can be removed rather than skipped. I'll push the changes shortly. @matzew